dolphin/task_manager/manager.py

```diff
@@ -66,6 +68,20 @@ def say_hello(self, context, request_spec=None,
         except Exception as ex:
             pass
 
+    def get_task_class(self, module, task, *args):
+        """
+        :param module:
+        :param task:
+        :param args:
+        :return:
+        """
+        return '{0}.{1}{2}({3})'.format(
```

Comment:
Put a comment here to tell what is the return. For readers to understand.
ex: resources.StoragePoolTask(storage_id)

Reply:
Added explanation with example